M2/Macaulay2/packages/FastLinAlg.m2

```diff
@@ -15,7 +15,7 @@ Version => "0.1", Date => "October 21st, 2019", Authors => {
     Email=> "yuhuiyao4ever@gmail.com"
     }
 }, --this file is in the public domain
-Headline => "faster linear algebra operations")
+Headline => "A package for faster linear algebra operations.", DebuggingMode => true, Reload=>true)
```

Comment:
Please leave DebuggingMode and Reload off.

Reply:
I've done this one, I'll reply as I've done the others.